### DISCUSSION TITLE
anyone using Mitosis with Panda CSS?

### DISCUSSION THREAD

**Discussion by alycda** [BuilderIO/jsx-lite#1331] [General]
https://panda-css.com/

**Comment by coopbri** [comment]
Hey, I have looked into doing this myself. I am maintaining a design system leveraging Panda CSS, Zag.js, and Ark UI and would love to integrate Mitosis instead of worrying about per-framework component bootstrapping. Asked a somewhat related question here: https://github.com/BuilderIO/mitosis/discussions/1349

I'll keep you posted with any progress I make!
